server/processors/src/test/java/org/hawkular/apm/server/processor/nodedetails/NodeDetailsDeriverTest.java

```diff
@@ -170,4 +170,67 @@ public void testInitial() throws RetryAttemptException {
         assertFalse(details.get(1).isInitial());
     }
 
+    @Test
+    public void testCalculateActualTimeSync() throws RetryAttemptException {
```

Comment:
Could you please add test, for which holds:

* one parent, one child 
* child duration < parent
* child ends after parent node.